action.yml

```
@@ -0,0 +1,72 @@
+inputs:
+  fetch-targets:
+    description: 'depviz target to fetch (format: \"org1/repo1 org1/repo2 org2 repo1\")'
+    default: '' 
+  generation-targets:
+    description: 'depviz target to generate outputs (format: \"org1/repo1 org1/repo2 org2 repo1\")' 
+    default: $GITHUB_REPOSITORY
```

```suggestion
inputs:
  targets:
    description: 'depviz target (format: \"org1/repo1 org1/repo2 org2 repo1\")'
    default: '%CURRENT_REPO%'
```

and `%CURRENT_REPO%` should be replaced in the targets so we can also make `org1/repo1 %CURRENT_REPO% org2/repo2`.

---

ok but why don't simply put `$GITHUB_REPOSITORY` instead of `%CURRENT_REPO%`